### PR TITLE
[OCPBUGS-18481] Add workaround required for BM workers with OSP on 4.11

### PR DESCRIPTION
**Pull request by MaysaMacedo**
Ticket: OCPBUGS-18481

Version(s):
4.11

Previews:
- https://66421--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom#installation-osp-deploying-bare-metal-machines_installing-openstack-installer-custom

- https://66421--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes#ocp-4-11-known-issues

QE review:
/cc @eurijon

### CODE REVIEW

File: modules/installation-osp-deploying-bare-metal-machines.adoc [hunk 1]
```diff
@@ -38,6 +38,8 @@ Be sure that your `install-config.yaml` file reflects whether the {rh-openstack}
 
 * If you want to deploy the machines on an installer-provisioned network, the {rh-openstack} Bare Metal service (Ironic) is able to listen for and interact with Preboot eXecution Environment (PXE) boot machines that run on tenant networks.
 
+* To workaround link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[known issue] in {rh-openstack}, customer has to include `force_metadata=False` in neutron-dhcp-agent configuration to enable nodes to access the metadata service.
```
Comment (maxwelldb) [inline modules/installation-osp-deploying-bare-metal-machines.adoc:41]:
```suggestion
* Due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[known issue] in {rh-openstack}, to give nodes access to the metadata service, you must set the value of `force_metadata` to `False` in the Neutron DHCP agent configuration.
```

**Comment by maxwelldb** [comment]
@MaysaMacedo This will need change management according to our DPM. 

The known issue should also be added to the 4.11 release notes.

**Comment by MaysaMacedo** [comment]
@maxwelldb I just added to the release notes, let me know what do you think of the section I chose to add it in.
Also, what is DPM?

**Comment by maxwelldb** [comment]
@MaysaMacedo Documentation Program Manager (Kathryn Alexander in OpenShift's case).

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -2515,6 +2515,8 @@ openshift.io/scc: hostmount-anyuid
 
 * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
 
+* When deploying a Bare Metal machine on {rh-openstack}, customers must set the value of `force_metadata` to `False` in the Neutron DHCP agent configuration to overcome a link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[known issue].
```
Comment (maxwelldb) [inline release_notes/ocp-4-11-release-notes.adoc:2518]: @JoeAldinger Assuming this makes it through change control, is this the appropriate place to add this item?

**Comment by eurijon** [comment]
Hey, this change looks good to me attending to @imatza-rh created OCPBUGS-18481 issue's description

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-11-release-notes.adoc:2518]: /lgtm

**Review by JoeAldinger — APPROVED** [review]

**Review by maxwelldb — APPROVED** [review]

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (maxwelldb) [inline release_notes/ocp-4-11-release-notes.adoc:2518]:
```suggestion
* When deploying a Bare Metal machine on {rh-openstack}, customers must set the value of `force_metadata` to `False` in the Neutron DHCP agent configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[*BZ2213862*])
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -2515,6 +2515,8 @@ openshift.io/scc: hostmount-anyuid
 
 * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
 
+* When deploying a Bare Metal machine on {rh-openstack}, customers must set the value of `force_metadata` to `False` in the Neutron DHCP agent configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[*BZ2213862*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-11-release-notes.adoc:2518]:
```suggestion
* When deploying a Bare Metal machine on {rh-openstack}, customers must set the value of the `force_metadata` option to `False` in the Neutron DHCP agent configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2213862[*BZ2213862*])
```

**Comment by tmalove** [comment]
@MaysaMacedo Be sure to squash your commits before requesting a merge.

/remove-label peer-review-in-progress
/label peer-review-done
/LGTM

**Comment by MaysaMacedo** [comment]
@maxwelldb @tmalove I just squashed it and applied Max suggestion

**Comment by maxwelldb** [comment]
@MaysaMacedo This needs change management. The remaining required acks are now requested on the Jira ticket.

**Review by mandre — COMMENTED** [review]
LGTM from a dev perspective.

**Comment by kalexand-rh** [comment]
@sferich888, will you please review this PR from a change management perspective? Stakeholders from the other functional areas have already approved it.

**Comment by kalexand-rh** [comment]
4.11 is now [EOL](https://access.redhat.com/support/policy/updates/openshift), so I'm closing this PR.

**Pull request closed without merging**